.travis.yml

```diff
@@ -1,16 +1,38 @@
-language: python
 sudo: required
```

Comment:
It's default now, you can remove this line.

Reply:
removed